simple_app/utils.py

```diff
@@ -1186,14 +1186,14 @@ def main_utils():
 
 
 REFERENCE_TABLES = [
```

Comment:
In the future, we could consider an import of SIMPLE so we can do `from simple import REFERENCE_TABLES` but we don't want to clone the repo just for that.    
Maybe adding it to the requirements in the form of a line like `simple @ git+https://github.com/SIMPLE-AstroDB/SIMPLE-db`?